### DISCUSSION TITLE
My NanoELS-build

### DISCUSSION THREAD

**Discussion by Hedning003** [kachurovskiy/nanoels#16] [Show and tell]
My attempt on this..have to do some tweaks to the .iso - made the bracket for the motor from some 8mm scrap-steel,and just put it in the mill- the electronic-box is NOT the final design,just had to mock something up to hold debree out-will make a proper one from a small steelcabinet,and attach it to the backside

![motorfäste](https://user-images.githubusercontent.com/95235260/144319281-8c04eb45-fe67-41fb-aa0d-1c6a96dad21b.jpg)
![panel](https://user-images.githubusercontent.com/95235260/144319287-d68426dc-bb5f-4d08-aafb-b13bf235e119.jpg)
![elektronik](https://user-images.githubusercontent.com/95235260/144319296-940636c7-76ef-47a4-9c89-acca3ce5b4a8.jpg)

**Comment by kachurovskiy** [comment]
Glad that you got it working! Nice way to get the motor bracket done, I assume an angle grinder and the drill press is all that was needed. Awesome!

**Comment by Hedning003** [comment]
Today i got some time for myself and made a proper electronic instalation ,i also tride the threadcutting on a piece of 30mm aluminium - seems to work great.   Thanks for the support on my questions,max - i am now a happy camper 
A note to other builders - check the movement properly - the 2mm leadscrew seems to differ some..in the iso-file i had to edit the 200-number for the lead to 197 to get my DRO and the ELS to measure the same

![elskap](https://user-images.githubusercontent.com/95235260/147705202-d473a812-22db-4b63-8f56-d75f11767aec.jpg)
![gänga](https://user-images.githubusercontent.com/95235260/147705203-c61029bb-55e5-44f4-aec3-8876d77c20f8.jpg)

**Reply by kachurovskiy** [reply]
Awesome, thank you!
